src/DevChatter.Bot.Core/Automation/IAutomatedActionSystem.cs

```diff
@@ -1,7 +1,12 @@
-﻿namespace DevChatter.Bot.Core.Automation
+using System.Collections.Generic;
+using DevChatter.Bot.Core.Data.Model;
+using DevChatter.Bot.Core.Systems.Chat;
+
+namespace DevChatter.Bot.Core.Automation
 {
     public interface IAutomatedActionSystem
     {
+        void AddAutomatedMessages(IEnumerable<IntervalMessage> messages, IList<IChatClient> chatClients);
```

Comment:
It seems a little "weird" to me that the automated action system has any idea what an IntervalMessage is given that IntervalMessage isn't an action. This looks to be little more than a "helper method" to transform a collection of IntervalMessages into `AutomatedMessage`s it might make more sense to create another Model extension like .ToAutomatedMessage and have BotMain register them as actions itself instead.